modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -1,54 +1,54 @@
 //Module included in the following assemblies:
 //
-//* microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+//* microshift_running_apps/microshift_operators/microshift-operators-oc-mirror.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-oc-mirror-list-operators-catalogs_{context}"]
 = Inspecting catalog contents by using the oc-mirror plugin
 
-Use the following example procedure to select a catalog and list Operators from available {OCP} content to add to your oc-mirror plugin image set configuration file.
+Use the following example procedure to select a catalog and list {OCP} Operators to add to your oc-mirror plugin image set configuration file. The following procedure uses oc mirror v1.
```

Comment:
Instead of saying `The following procedure uses oc mirror v1` can we say select a catalog and list {OCP} Operators can currently be done only via oc-mirror v1 and below procedure explains how to do that.